webui/react/src/pages/Settings/UserManagement.test.tsx

```diff
@@ -20,6 +20,7 @@ const DISPLAY_NAME = 'Test Name';
 const USERNAME = 'test_username1';
 
 const user = userEvent.setup();
+jest.useRealTimers();
```

Comment:
the change might stop flakiness, but the current test case doesnt capture the modal according to [this](https://stackoverflow.com/questions/68829540/react-testing-library-testing-ant-design-modal/68843345#68843345). 
so i think we need to fix the original issue instead of hiding it
also some waitFor missing `await`, so there might be flaky test cases somewhere else.

Reply:
could you add `await` before `waitFor` in this file?
it doesn't properly working without it

Reply:
flaky test is fixed in #5990, so i think we can remove `jest.useRealTimers();`